defaults.gpr

```diff
@@ -27,6 +27,14 @@ abstract project Defaults is
       when "asserts_enabled" | "runtime_compatible" =>
    end case;
 
+   Global_Configuration_Pragmas := "defaults_backward_compatible.adc";
```

Comment:
Couldn't this better go into the empty `others =>` branch below?

Reply:
I had this before, but a variable cannot be declared inside a `case` statement. An option would be declaring it with an empty string first. Would you prefer that?

Reply:
No, better leave it as is then.